### PR TITLE
feat: new docs about mtg, sdk and listing assets at 4swap

### PR DESCRIPTION
**Pull request by lyricat**

### CODE REVIEW

File: docs/lake/key-concepts/4swap-mtg.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: 4swap MTG
+sidebar_position: 5
+date: 2021-08-15 22:33:07
+---
+
+4swap is built with [Mixin Trusted Group](https://developers.mixin.one/document/mainnet/mtg). Right now, there are 5 genesis nodes who serve the trades, they are:
```
Comment (YaruiP) [inline docs/lake/key-concepts/4swap-mtg.md:7]: "there are 5 genesis nodes who serve the trades, they are: " --> " there are 5 managerial nodes that performs administrative actions. They are: "

File: docs/lake/key-concepts/4swap-mtg.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+- [Mixin Team](https://mixin.one/): The core develop team of Mixin Network
```
Comment (YaruiP) [inline docs/lake/key-concepts/4swap-mtg.md:9]: "develop team" --> "development team"

File: docs/lake/key-concepts/4swap-mtg.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+- [Fox.ONE](https://fox.one/): The main develop team of 4swap
```
Comment (YaruiP) [inline docs/lake/key-concepts/4swap-mtg.md:10]: --> "development team"

File: docs/lake/key-concepts/4swap-mtg.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+- [Poolin](https://poolin.com/): The industry-leading mining service provider
+- [Big.ONE](https://big.one/): A trading platform provides users with the secure and trusteeship services
```
Comment (YaruiP) [inline docs/lake/key-concepts/4swap-mtg.md:12]: --> "A **global** trading platform **that** provides users with **secure and convenient** trusteeship services

File: docs/lake/tutorials/listing.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+---
+title: Listing assets and adding pools
+sidebar_position: 6
+date: 2021-08-15 22:33:07
+---
+
+**[Pando Lake](https://lake.pando.im)** only allow for selected pools that have proven credentials. It not accept new pool from the public.
```
Comment (YaruiP) [inline docs/lake/tutorials/listing.md:7]: Pando Lake only **allows** for... 

"It not accept" --> "It doesn't accept..."

File: docs/lake/tutorials/listing.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+For example, if you have a ERC-20 token named "RUM", you can add it to Mixin Network:
+
+1. an icon file named "icon.png", it should be 520px * 520px.
+2. a json file named "index.json", which should be a valid json file and contain the following information:
+
+    ```json
+    {
+      // the asset id of RUM in Mixin Network
+      "asset_id": "4f2ec12c-22f4-3a9e-b757-c84b6415ea8f",
+      // RUM is a ERC-20 token, the chain id is ETH's asset id
```
Comment (YaruiP) [inline docs/lake/tutorials/listing.md:24]: **"an** ERC-20 token"

File: docs/lake/tutorials/listing.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    ```
+The commit of the above example is [here](https://github.com/MixinNetwork/asset-profile/commit/437d378f899c5837598bdb8c4e9c18ae8f21ad27).
+
+:::info
+Please create a pull request when you have finished. It usually takes about 1~2 days for Mixin Network to process your pull request.
+:::
+
+## Create a pool at 4swap
+
+You need to Mixin Messenger or Fennec to create a pool.
```
Comment (YaruiP) [inline docs/lake/tutorials/listing.md:38]: need to **head** to Mixin...

File: docs/lake/tutorials/listing.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+1. Make sure the 2 assets in your wallet balance is above the minimum amount of 0.00000001. If not, please deposit some to your wallet.
+2. Make sure you have above the minimum amount of 0.02 ETH in your wallet. If not, please deposit some into your wallet or swap for it at [4swap](https://app.4swap.org).
+3. Visit [this page](https://app.4swap.org/#/me), tap "connect" to connect with your wallet.
+4. Visit [this page](https://app.4swap.org/#/liquidity/create), choose two assets you want to add to the pool and tap `Pay` to create a pool.
+
+:::info
+It usually cost 1~3 days to wait for the pool to be created and approved by [MTG nodes](../key-concepts/4swap-mtg). If the request is not processed in 3 days, you will receive a refund.
```
Comment (YaruiP) [inline docs/lake/tutorials/listing.md:46]: It usually **takes** 1～3 days **for the pool to be approved and created**  by...

File: docs/leaf/key-concepts/leaf-mtg.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
+---
+title: Pando Leaf MTG
+sidebar_position: 4
+date: 2021-08-15 22:33:07
+---
+
+Pando Leaf is built with [Mixin Trusted Group](https://developers.mixin.one/document/mainnet/mtg). Right now, there are 6 genesis nodes who serve the trades, they are:
```
Comment (YaruiP) [inline docs/leaf/key-concepts/leaf-mtg.md:7]: -->" ... there are 6 managerial nodes that performs administrative actions. They are: "

File: docs/leaf/key-concepts/leaf-mtg.md [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+
+- [Mixin Team](https://mixin.one/): The core develop team of Mixin Network
+- [Fox.ONE](https://fox.one/): The main develop team of 4swap
+- [Poolin](https://poolin.com/): The industry-leading mining service provider
+- [Big.ONE](https://big.one/): A trading platform provides users with the secure and trusteeship services
```
Comment (YaruiP) [inline docs/leaf/key-concepts/leaf-mtg.md:12]: --> "A global trading platform that provides users with secure and convenient trusteeship services

File: docs/leaf/key-concepts/leaf-mtg.md [hunk 11]
```diff
@@ -0,0 +1,14 @@
+---
+title: Pando Leaf MTG
+sidebar_position: 4
+date: 2021-08-15 22:33:07
+---
+
+Pando Leaf is built with [Mixin Trusted Group](https://developers.mixin.one/document/mainnet/mtg). Right now, there are 6 genesis nodes who serve the trades, they are:
+
+- [Mixin Team](https://mixin.one/): The core develop team of Mixin Network
```
Comment (YaruiP) [inline docs/leaf/key-concepts/leaf-mtg.md:9]: develop --> development

File: docs/leaf/key-concepts/leaf-mtg.md [hunk 12]
```diff
@@ -0,0 +1,14 @@
...
+- [Fox.ONE](https://fox.one/): The main develop team of 4swap
```
Comment (YaruiP) [inline docs/leaf/key-concepts/leaf-mtg.md:10]: develop --> development

**Review by YaruiP — COMMENTED** [review]
Few edits

File: docs/lake/key-concepts/4swap-mtg.md (hunk 1 above)
Comment (lyricat) [reply docs/lake/key-concepts/4swap-mtg.md:7]: genesis nodes 的表述没错哦。

File: docs/lake/key-concepts/4swap-mtg.md (hunk 3 above)
Comment (lyricat) [reply docs/lake/key-concepts/4swap-mtg.md:10]: 好

File: docs/lake/key-concepts/4swap-mtg.md (hunk 1 above)
Comment (YaruiP) [reply docs/lake/key-concepts/4swap-mtg.md:7]: 我看bitcoin genesis block是指创始区块(https://www.investopedia.com/terms/g/genesis-block.asp)，有可以往上增加的意思。我们这五个节点，主要表达是管理责任，用genesis nodes 会不会可能引起误解捏

**Pull request merged**